client/packages/system/src/Patient/api/hooks/document/usePatients.ts

```diff
@@ -1,15 +1,17 @@
 import { useQuery, useUrlQueryParams } from '@openmsupply-client/common';
+import { ListParams } from '../../api';
 import { usePatientApi } from '../utils/usePatientApi';
 
-export const usePatients = () => {
+export const usePatients = (customQuery?: ListParams) => {
   const api = usePatientApi();
   const { queryParams } = useUrlQueryParams({
     filterKey: ['lastName', 'firstName', 'identifier'],
     initialSort: { key: 'code', dir: 'asc' },
   });
+
+  const query = customQuery ?? queryParams;
```

Comment:
Needed to be able to add query params beyond what is in the URL.

Reply:
Related comment, I found the hooks sometimes confusing because its hard to figure out where the hook is getting its params from and thus where the hook can be used. Find it easier to read when the params are passed in explicitly...

Reply:
Not sure what you're referring to. You mean in this case, or for the database hooks in general?

Reply:
No was a general comment for hooks where it might be simpler to pass in the params rather than implicitly fetch them from the url...